deployment/terraform/instances/shared/README.md

```diff
@@ -0,0 +1,7 @@
+This stack contains the ECR repository definition in the shared account.
+
+Deployment is done locally via terraform - ensure you are authenticated with the `emnify-shared` account 
```

Comment:
Out of curiosity - where can I read more about emnify-shared account? What's it for, how it's managed etc.

Reply:
@zabbal I think the best guy to explain to you is @StephenKing